.github/workflows/sphinx.yml

```diff
@@ -0,0 +1,52 @@
+name: sphinx
```

Comment:
How about calling the workflow (and the file) `documentation(.py)`?

Reply:
84a57270ac524635505cf13e3a747912696f996b